### PR TITLE
Update modern-auth-for-office-2013-and-2016.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reflect office 2019 as well

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/941

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ description: "Learn how Microsoft 365 modern authentication works differently fo
 
 *This article applies to both Microsoft 365 Enterprise and Office 365 Enterprise.*
 
-Read this article to learn how Office 2013 and Office 2016 client apps use modern authentication features based on the authentication configuration on the Microsoft 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
+Read this article to learn how Office 2013, Office 2016 and Office 2019 client apps use modern authentication features based on the authentication configuration on the Microsoft 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:32]:
```suggestion
Read this article to learn how Office 2013, Office 2016, and Office 2019 client apps use modern authentication features based on the authentication configuration on the Microsoft 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ Read [How to use Modern Authentication (ADAL) with Skype for Business](https://g
   
 Office 2016 and Office 2019  clients support modern authentication by default, and no action is needed for the client to use these new flows. However, explicit action is needed to use legacy authentication.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:63]:
```suggestion
Office 2016 and Office 2019 clients support modern authentication by default, and no action is needed for the client to use these new flows. However, explicit action is needed to use legacy authentication.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Read [How to use Modern Authentication (ADAL) with Skype for Business](https://g
...
   
-Click the links below to see how Office 2013 and Office 2016 client authentication works with the Microsoft 365 services depending on whether or not modern authentication is turned on.
+Click the links below to see how Office 2013, Office 2016 and Office 2019 client authentication works with the Microsoft 365 services depending on whether or not modern authentication is turned on.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:65]:
```suggestion
Click the links below to see how Office 2013, Office 2016, and Office 2019 client authentication works with the Microsoft 365 services depending on whether or not modern authentication is turned on.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 4]
```diff
@@ -73,15 +73,15 @@ Click the links below to see how Office 2013 and Office 2016 client authenticati
 <a name="BK_EchangeOnline"> </a>
 ### Exchange Online
 
-The following table describes the authentication behavior for Office 2013 or Office 2016 client apps when they connect to Exchange Online with or without modern authentication.
+The following table describes the authentication behavior for Office 2013. Office 2016 and Office 2019 client apps when they connect to Exchange Online with or without modern authentication.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:76]:
```suggestion
The following table describes the authentication behavior for Office 2013, Office 2016, and Office 2019 client apps when they connect to Exchange Online with or without modern authentication.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 5]
```diff
@@ -91,7 +91,7 @@ The following table describes the authentication behavior for Office 2013 or Off
 <a name="BK_SharePointOnline"> </a>
 ### SharePoint Online
 
-The following table describes the authentication behavior for Office 2013 or Office 2016 client apps when they connect to SharePoint Online with or without modern authentication.
+The following table describes the authentication behavior for Office 2013, Office 2016 and Office 2019 client apps when they connect to SharePoint Online with or without modern authentication.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:94]:
```suggestion
The following table describes the authentication behavior for Office 2013, Office 2016, and Office 2019 client apps when they connect to SharePoint Online with or without modern authentication.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 6]
```diff
@@ -107,7 +107,7 @@ The following table describes the authentication behavior for Office 2013 or Off
 ### Skype for Business Online
 <a name="BK_SFBO"> </a>
 
-The following table describes the authentication behavior for Office 2013 or Office 2016 client apps when they connect to Skype for Business Online with or without modern authentication.
+The following table describes the authentication behavior for Office 2013, Office 2016 and Office 2019 client apps when they connect to Skype for Business Online with or without modern authentication.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:110]:
```suggestion
The following table describes the authentication behavior for Office 2013, Office 2016, and Office 2019 client apps when they connect to Skype for Business Online with or without modern authentication.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 7]
```diff
@@ -73,15 +73,15 @@ Click the links below to see how Office 2013 and Office 2016 client authenticati
...
+The following table describes the authentication behavior for Office 2013. Office 2016 and Office 2019 client apps when they connect to Exchange Online with or without modern authentication.
   
 |****Office client app version****|****Registry key present?****|****Modern authentication on?****|****Authentication behavior with modern authentication turned on for the tenant (default)****|****Authentication behavior with modern authentication turned off for the tenant****|
 |:-----|:-----|:-----|:-----|:-----|
-|Office 2019  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013, or 2019 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
+|Office 2019  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2013, 2016, or 2019 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:80]:
```suggestion
|Office 2019  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2013, 2016, or 2019. <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 8]
```diff
@@ -73,15 +73,15 @@ Click the links below to see how Office 2013 and Office 2016 client authenticati
...
 |Office 2019  <br/> |Yes, EnableADAL=0  <br/> |No  <br/> |Basic authentication  <br/> |Basic authentication  <br/> |
-|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013, or 2016 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
+|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on 2013, 2016, or 2019 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:84]:
```suggestion
|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on 2013, 2016, or 2019. <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**